code/game/objects/items/weapons/material/swords.dm

```diff
@@ -65,3 +67,15 @@
 	icon_state = "sabre"
 	item_state = "katana"
 	slot_flags = SLOT_BELT
+	
+/obj/item/weapon/material/sword/axe
+	name = "battle axe"
+	desc = "An one handed battle axe, still a deadly weapon."

```

Comment:
Actually. Hate to be a downer, but it's still "A one handed battle axe." The article is dependent on the noun it's associated with (battle axe), not the adjectives (one handed) that come before it.